### PR TITLE
Removed hard coded phrases

### PR DESCRIPTION
**Pull request by mockdot**
Removed hard coded phrases from latest PR's, as well as the blip name which was missing

### CODE REVIEW

File: locales/es.lua [hunk 1]
```diff
@@ -9,14 +9,20 @@ local Translations = {
     },
     info = {
         ["refuel_vehicle"] = "Rellenar vehículo",
-        ["gas_station"] = "Estación de gasolina",
+        ["take_nozzle"] = "Tomar la boquilla",
+        ["return_nozzle"] = "Devolver la boquilla",
+        ["gas_station"] = "Estación de servicio",
```
Comment (MonkeyWhisper) [inline locales/es.lua:14]: Put this back as

  ["gas_station"] = "Estación de gasolina",

**Review by MonkeyWhisper — CHANGES_REQUESTED** [review]

File: locales/es.lua (hunk 1 above)
Comment (mockdot) [reply locales/es.lua:14]: Why tho? In all spanish countries we do not call it "Estacion de gasolina", it's an "Estación de servicio". "Estación de gasolina" is a google translation, not a real one.

Comment (mockdot) [reply locales/es.lua:14]: https://es.wikipedia.org/wiki/Estaci%C3%B3n_de_servicio

Comment (MonkeyWhisper) [reply locales/es.lua:14]: I speak Spanish and and the correct term is Estacion de gasolina, if you want this PR merged this change needs to take effect else, feel free to close the PR.

Comment (mockdot) [reply locales/es.lua:14]: No, it's not. I live in Costa Rica and have lived in Argentina, Venezuela, México, and the US even. That's not the term used. You already closed it yourself, I won't be submitting other PR's, but thanks for the douchy attitude.

Comment (MonkeyWhisper) [reply locales/es.lua:14]: That's fine, we don't want wrong PRs that are incorrect coming from you. 

Have a great weekend!

**Pull request closed without merging**